x-pack/test/security_api_integration/tests/session_concurrent_limit/cleanup.ts

```diff
@@ -240,12 +240,30 @@ export default function ({ getService }: FtrProviderContext) {
       log.debug(`Log in as ${testUser.username} and SAML user 3 times each with a 0.5s delay.`);
 
       const basicSessionCookieOne = await loginWithBasic(testUser);
+      await retry.tryForTime(20000, async () => {
+        expect(await getNumberOfSessionDocuments()).to.be(1);
+      });
+      await setTimeoutAsync(500);
```

Comment:
Do we need to keep the delays if we're adding a retry with timeout after each login?

Reply:
Good point! Removed!